Resources/public/js/components/price-calculation-util/price-calculation-util.js

```diff
@@ -0,0 +1,349 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define([], function() {
```

Comment:
a question about the path of this file:
```
Resources/public/js/components/price-calculation-util/price-calculation-util.js
```

since this is not a aura-component, i'd prefer another location for the file:
what about 
```
Resources/public/js/util/price-calculation-util.js
```
?